### PR TITLE
Update front page, stressing open source and open development

### PR DESCRIPTION
**Pull request by dpiparo**

### CODE REVIEW

**Comment by dpiparo** [comment]
image

File: index.html [hunk 1]
```diff
@@ -12,10 +12,10 @@
...
       url: /install
 feature_row:
-  - title: '<img src="assets/images/histoIcon.svg" alt="*.*" width="59" height="59">'
+  - title: '<img src="assets/images/histoIcon.svg" alt="*.*" width="65" height="65">'
     excerpt: "ROOT enables <i>statistically sound</i> scientific analyses and visualization of large amounts of data: today, more than 1 exabyte (1,000,000,000 gigabyte) are stored in ROOT files. [The Higgs was found with ROOT!](gallery#higgs-plots)"
-  - title: '<img src="assets/images/globe_with_meridians.svg" alt="*.*">'
-    excerpt: "As <i>high-performance</i> software, ROOT is written mainly in C++. You can use it on Linux, macOS, or Windows; it works out of the box. ROOT is [open source](about/license): use it freely, [modify it](install/build_from_source), [contribute to it](contribute)!"
+  - title: '<img src="assets/images/openHand.svg" alt="*.*" width="65" height="65">'
+    excerpt: "ROOT is [open source](about/license), which means that you can use it freely and [modify it](install/build_from_source), and adopts an open development process, inviting [its users to contribute to it](contribute)"
```
Comment (hageboeck) [inline index.html:18]:
```suggestion
    excerpt: "ROOT is [open source](about/license), which means that you can use it freely and [modify it](install/build_from_source). It adopts an open development process, inviting [its users to contribute to it](contribute)"
```

**Review by hageboeck — COMMENTED** [review]
I would suggest to split the sentence.

File: index.html (hunk 1 above)
Comment (dpiparo) [reply index.html]: Thanks. Done.

**Pull request merged**